workers/data_refinery_workers/downloaders/utils.py

```diff
@@ -0,0 +1,20 @@
+from django.utils import timezone
+from data_refinery_models.models import Batch, BatchStatuses, DownloaderJob
+
+
+def start_job(job: DownloaderJob):
+    """Record in the database that this job is being started. """
+    job.worker_id = "For now there's only one. For now..."
```

Comment:
Where will we get these worker_ids from in the future? Are they automatically assigned by Celery?